tools/windows/DatadogAgentInstaller/WixSetup/localization-en-us.wxl

```diff
@@ -1,61 +1,69 @@
+	<String Id ="ApiKeyDialogKeyLabel" Overridable="yes" Localizable="yes">Please enter the Datadog API Key:</String>
+	<String Id ="ApiKeySampleKey" Overridable="yes" Localizable="yes"></String>
+
+	<!-- these strings for the  site dialog -->
+	<String Id ="SiteDialog_Title" Overridable="yes" Localizable="yes">[ProductName] Setup</String>
+	<String Id ="SiteDialogDescription" Overridable="yes" Localizable="yes">{\WixUI_Font_Normal_White}Select the Datadog Region</String>
+	<String Id ="SiteDialogTitle"  Overridable="yes" Localizable="yes">{\WixUI_Font_Title_White}Datadog Region </String>
+	<String Id ="SiteDialogKeyLabel" Overridable="yes" Localizable="yes">Which Datadog region do you want the Agent to send data to?</String>
+	<String Id ="SiteSampleKey" Overridable="yes" Localizable="yes"></String>
+
+	<!-- these strings for the agent user dialog -->
+	<String Id ="DDAgentUserDialog_Title" Overridable="yes" Localizable="yes">[ProductName] Setup</String>
+	<String Id ="DDAgentUserDialogDescription" Overridable="yes" Localizable="yes">{\WixUI_Font_Normal_White}Enter the agent user account</String>
+	<String Id ="DDAgentUserDialogTitle"  Overridable="yes" Localizable="yes">{\WixUI_Font_Title_White}Datadog Agent User Account</String>
+	<String Id ="DDAgentUserDialogUserNameLabel" Overridable="yes" Localizable="yes">Please enter the username (leave blank to use ddagentuser):</String>
+	<String Id ="DDAgentUserDialogPasswordLabel" Overridable="yes" Localizable="yes">Please enter the password (leave blank to use random password):</String>
+
+	<!-- these strings for the FatalError dialog -->
+	<String Id ="FatalErrorDialog_ViewLog" Overridable="yes" Localizable="yes">View Log</String>
+	<String Id ="FatalErrorDialog_SendFlare" Overridable="yes" Localizable="yes">Send Flare</String>
+	<String Id ="FatalErrorDialog_ViewLogDescription" Overridable="yes" Localizable="yes">Detailed information on the error can be found in the installation log file. Click the View Log button to view the installation log. Click the Send Flare button to send a diagnostic report of this error to Datadog.</String>
+
+	<!-- these strings for the send flare dialog -->
+	<String Id ="SendFlareDialog_Title" Overridable="yes" Localizable="yes">[ProductName] Setup</String>
+	<String Id ="SendFlareDialogDescription" Overridable="yes" Localizable="yes">{\WixUI_Font_Normal_White}Send a diagnostic report of this installation error to Datadog.</String>
+	<String Id ="SendFlareDialogTitle"  Overridable="yes" Localizable="yes">{\WixUI_Font_Title_White}Datadog Flare </String>
+	<String Id ="SendFlareDialogKeyLabel" Overridable="yes" Localizable="yes">Clicking on the Send Flare button will send a diagnostic report to Datadog, including the install logs.</String>
+	<String Id ="SendFlareHelpText" Overridable="yes" Localizable="yes">If you like you can specify your e-mail address in the text box below and a member of our support staff will follow up with you.</String>
```

Comment:
small suggestion for text: 

> If you like you can specify your e-mail address in the text box below and a member of our support staff will follow up with you

"Submit your email address in the text box below for a follow-up from Datadog support. (Optional)"